CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ set_target_properties(lua::lua
     IMPORTED_LINK_DEPENDENT_LIBRARIES ${LIBM_LIB}
     )
 
+set(LUA_FOUND TRUE)
```

Comment:
@aderiabin that if lua is not on the system, will still be true?

Reply:
@LuxoftAKutsan In case appropriate Lua version is not found in system cmake will be failed, so LUA_FOUND will be set only in case of successful Lua version find.

This flag was added for support separate building of subproject `remote_adapter`

Reply:
@aderiabin Try next:
if (NOT TARGET lua::lua)
add_library(lua::lua ....) # create imported target
# set target props you need
endif()

Reply:
@EKuliiev Applied (see c09eab5)
Thank you.